### PR TITLE
TOTAL. IMPULSE. DEATH.

### PR DESCRIPTION
**Pull request by InvalidArgument3**

### CODE REVIEW

**Comment by Jnick-24** [comment]
i wish i were as cool as this commit

**Review by Jnick-24 — CHANGES_REQUESTED** [review]
wow this looks good except for the 1 mass

# **TOTAL. IMPULSE. DEATH.**

**Comment by CommodoreChet** [comment]
Revert Changes made to NHI before Merging this.

**Comment by Tiberias42** [comment]
Way to hard to read on my phone, will get on when I'm home from work.

**Comment by BadM0dder** [comment]
Hah funny meme I'll just revert tio changes next pull :whenstarcore:

**Comment by TaitKaito** [comment]
Send it for Fletcher boys, TOTAL IMPULSE DEATH LESSGOOOO

**Review by Tiberias42 — APPROVED** [review]
I looked at the Serpent Arms.  I'm sure we have a perfectly good reason to make every bullet that had no weight at all 1 kg.  I am ok with this for SA.

**Comment by Jnick-24** [comment]
Okay, just as a personal reference - what guns would you guys consider to "need" impulse? Aries, hunter pan, etc?

File: Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Experimental/EXP_BFG_Ammo_Cannon_Ammo.cs
Comment (CommodoreChet) [inline Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Experimental/EXP_BFG_Ammo_Cannon_Ammo.cs:1]: Revert - Not on Active Roster

File: Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Fixed/NHI_Fixed_Torpedo_Launcher_Ammo.cs
Comment (CommodoreChet) [inline Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Fixed/NHI_Fixed_Torpedo_Launcher_Ammo.cs:1]: Revert - Not on Active Roster

File: Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Starcore/SC_Coil_Cannon_Ammo.cs
Comment (CommodoreChet) [inline Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Starcore/SC_Coil_Cannon_Ammo.cs:1]: Revert
Changes Already Planned to Correct Shield Behavior

File: Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Flak_Cannon_Turret_Ammo.cs
Comment (CommodoreChet) [inline Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Flak_Cannon_Turret_Ammo.cs:1]: Revert - Not on Active Roster

File: Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Heavy_Gun_Turret_Ammo.cs
Comment (CommodoreChet) [inline Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Heavy_Gun_Turret_Ammo.cs:1]: Revert
Mass on Primary Round is irrelevant as the round is Prox Fused. Change Unnecessary.

File: Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Heavy_Railgun_Turret_Ammo.cs
Comment (CommodoreChet) [inline Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Heavy_Railgun_Turret_Ammo.cs:1]: Revert - Not on Active Roster

File: Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Kinetic_Cannon_Turret_Ammo.cs
Comment (CommodoreChet) [inline Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Kinetic_Cannon_Turret_Ammo.cs:1]: Revert
This Round was Properly Balanced and has not shown any Issues

File: Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Light_Autocannon_Turret_Ammo.cs
Comment (CommodoreChet) [inline Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Light_Autocannon_Turret_Ammo.cs:1]: Revert
This Round was Properly Balanced and has not shown any Issues

File: Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Light_Railgun_Turret_Ammo.cs
Comment (CommodoreChet) [inline Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Light_Railgun_Turret_Ammo.cs:1]: Revert
This Round was Properly Balanced and has not shown any Issues

File: Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Mk1_Cannon_Turret_Ammo.cs
Comment (CommodoreChet) [inline Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Mk1_Cannon_Turret_Ammo.cs:1]: Revert
This Round was Properly Balanced and has not shown any Issues
There are Misc. Disabled Ammos in this File that should not be altered.

File: Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Mk2_Cannon_Turret_Ammo.cs
Comment (CommodoreChet) [inline Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Mk2_Cannon_Turret_Ammo.cs:1]: Revert
This Round was Properly Balanced and has not shown any Issues
There are Misc. Disabled Ammos in this File that should not be altered.

File: Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Mk3_Cannon_Turret_Ammo.cs
Comment (CommodoreChet) [inline Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_Mk3_Cannon_Turret_Ammo.cs:1]: Revert
This Round was Properly Balanced and has not shown any Issues
There are Misc. Disabled Ammos in this File that should not be altered.

File: Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_PD_Turret_Ammo.cs
Comment (CommodoreChet) [inline Weapon Mods/Stable/NorseHeavyIndustries/Data/Scripts/Coreparts/NHI_Turreted/NHI_PD_Turret_Ammo.cs:1]: Revert
Round is Prox Fused and does not target Large Grids | Mass is not relevant to Starcore Balance, Required for Internal Consistency

**Review by CommodoreChet — CHANGES_REQUESTED** [review]

**Comment by InvalidArgument3** [comment]
wait fuck

**Comment by InvalidArgument3** [comment]
not gonna modify 300 cs files for this

**Pull request closed without merging**